gapic/ads-templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2

```diff
@@ -77,6 +84,8 @@ def test_{{ service.client_name|snake_case }}_get_transport_class():
     assert transport == transports.{{ service.name }}GrpcTransport
 
 
+@mock.patch.object({{ service.client_name }}, "DEFAULT_ENDPOINT", modify_default_endpoint({{ service.client_name }}))
+@mock.patch.object({{ service.async_client_name }}, "DEFAULT_ENDPOINT", modify_default_endpoint({{ service.async_client_name }}))
```

Comment:
I don't think the async client has been ported to the ads templates.

Reply:
removed